code/modules/client/preference/preferences.dm

```diff
@@ -2035,6 +2038,10 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "wingdings"
```

Comment:
If I moved it completely to `on_species_change`, would it still apply to first spawning, or should it copypasta over?